template.rb

```diff
@@ -1,6 +1,6 @@
 # Hack for https://github.com/rails/rails/issues/35153
 gsub_file 'Gemfile', /^gem ["']sqlite3["']$/, 'gem "sqlite3", "~> 1.3.0"'
-gem 'hyrax', '3.0.0-beta2'
+gem 'hyrax', '3.0.0-beta3', path: '~/src/hyrax'
```

Comment:
I think you should remove the path part here.  Probably vestigial from testing.

Reply:
thanks! good catch